### PR TITLE
[FIX] Fixing error External ID not found on country from on click button

### PR DESCRIPTION
**Pull request by valentinagiuliani-rapsodoo**
[FIX] Fixing error "External ID not found in the system: product_country_restriction.product_country_restrictionact_window" on click button on country form

### CODE REVIEW

**Review by Gonzo1795 — APPROVED** [review]

**Review by simoneversienti — APPROVED** [review]

**Comment by simoneversienti** [comment]
:+1:

**Review by OpenCode — APPROVED** [review]

**Review by acorpuz — APPROVED** [review]

File: product_country_restriction/models/res_country.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def _compute_product_country_restriction_ids(self):
 
     def action_view_country_restrictions(self):
         self.ensure_one()
-        ref = "product_country_restriction." "product_country_restrictionact_window"
+        ref = "product_country_restriction." "product_country_restriction_act_window"
```
Comment (rousseldenis) [inline product_country_restriction/models/res_country.py]:
```suggestion
        action_dict = self.env["ir.actions.act_window"]._for_xml_id("product_country_restriction.product_country_restriction_act_window")
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@valentinagiuliani-rapsodoo Could you check pre-commit ?

**Comment by rousseldenis** [comment]
@valentinagiuliani-rapsodoo Could you merge the pre-commit commit with the preceding one?

**Comment by rousseldenis** [comment]
@valentinagiuliani-rapsodoo Last thing : could you modify your commit messages to reflect module name like `[IMP] product_country_restirction: (...)`? Thanks

**Review by OpenCode — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**